src/main/app/src/app/zaken/zaken-werkvoorraad/zaken-werkvoorraad.component.ts

```diff
@@ -175,7 +175,7 @@ export class ZakenWerkvoorraadComponent implements AfterViewInit, OnInit, OnDest
     }
 
     showAssignToMe(row: ZaakZoekObject): boolean {
-        return this.ingelogdeMedewerker.id !== row.behandelaarGebruikersnaam;
+        return this.ingelogdeMedewerker && this.ingelogdeMedewerker.id !== row.behandelaarGebruikersnaam;
```

Comment:
Hoe kan het dat er geen ingelogde medewerker is ??? In dat geval zou er toch veel meer mis moeten gaan?